docs/dispatcher/router.rst

```diff
@@ -232,7 +232,7 @@ Example:
 
 .. code-block:: python
     :caption: module_1.py
-:name: module_1
+    # module_1
```

Comment:
The problem you described is caused by incorrect indentation, so you should simply add 4 spaces before the `:name:` ​​property instead of replacing it with a comment.